lib/gui/components/safe-webview/safe-webview.js

```diff
@@ -167,13 +167,9 @@ class SafeWebview extends react.PureComponent {
    * @param {Event} event - Event object
    */
   didGetResponseDetails(event) {
-
-    const {
-      httpResponseCode
-    } = event;
     const HTTP_OK = 200;
 
-    if (httpResponseCode !== HTTP_OK) {
+    if (event.httpResponseCode !== HTTP_OK) {
       this.setState({
         shouldLoad: false
       });
```

Comment:
Does this happen for just the first load, or for all page-loads? i.e. if a user clicks on a link which for some reason is invalid or non-responsive, will the "default banner" get shown again after two seconds?